wp-content/themes/thinkery/style.css

```diff
@@ -1485,23 +1486,68 @@ p.entry-meta {
 	margin-bottom: 0;
 }
 
+.footer-widgets div ul li a {
+	color: white;
+	font-family: museo-sans, sans-serif;
+	font-size: .85em;
+	font-weight: 400;
+	text-decoration: none;
+}
+.footer-widgets #fa-icons {
+display: flex;
+justify-content: space-between;
+font-size: 1.5em;
+
+}
+
+.footer-widgets #fa-icons i:focus,
+.footer-widgets #fa-icons i:hover {
+	color: #1b2432;
+}
+
+.footer-widgets a:focus,
+.footer-widgets a:hover {
+	color: #1b2432;
+}
+
+.footer-widgets h3 {
+	color: white;
+  font-family: museo-sans, sans-serif;
+}
 
+.footer-widgets p {
+	color: white;
+	font-size: .85em;
+  font-family: museo-sans, sans-serif;
```

Comment:
same, except for font-size everything else is defined globally.

Reply:
Removed this unnecessary code, tested all still functioning as intended.